appserver/payara-appserver-modules/microprofile/openapi/src/main/java/fish/payara/microprofile/openapi/impl/model/media/ContentImpl.java

```diff
@@ -88,42 +120,49 @@ public void setMediaTypes(Map<String, MediaType> mediaTypes) {
         putAll(mediaTypes);
     }
 
-    public static void merge(org.eclipse.microprofile.openapi.annotations.media.Content from, Content to,
-            boolean override, Map<String, Schema> currentSchemas) {
+    public static void merge(Content from, Content to,
```

Comment:
If this method only works with `ContentImpl` passed it maybe should demand it in is signature? If `createInstance` would return `ContentImpl` the caller joining the two maybe has all necessary type data?